### ISSUE TITLE
Suggestion - make this work for a c# console app

### ISSUE THREAD

**Issue by dcleblanc** [bemehiser/cmake_csharp_wpf_example#2]
Nice and simple, HelloWorld.cs

What would be really cool is making it work on Linux with .NET Core

**Comment by bemehiser** [comment]
@dcleblanc, I agree it would be cool if it worked on Linux, but sadly (as of the last time I checked) the Visual Studio CMake generator isn't supported for Linux, so it's not possible.

**Comment by dcleblanc** [comment]
If you started with dotnet add, that will make the csproj file for you. If I could then sort out how to get cmake to do something which would then invoke dotnet build on this, it would be great. Been wrestling with this for some time, your code is the closest thing I've found.
